src/main/routes.ts

```diff
@@ -9,6 +9,15 @@ export default function(app: Application): void {
   app.get('/courts', app.locals.container.cradle.courtsController.get);
   app.get('/courts/download', app.locals.container.cradle.courtsDownloadController.get);
 
+  //Account
+  app.get('/users',app.locals.container.cradle.accountController.get);
+  app.get('/users/invite/user', app.locals.container.cradle.inviteUserController.renderUserInvite);
+  app.post('/users/invite/user', app.locals.container.cradle.inviteUserController.postUserInvite);
```

Comment:
Do we need to specify these as super admin only endpoints? i.e 

app.post('/users/invite/user', isSuperAdmin, app.locals.container.cradle.inviteUserController.postUserInvite);